src/ticket/ticket.service.ts

```diff
@@ -203,6 +204,63 @@ export class TicketService {
     }
   }
 
+  // Get all tickets belonging to a person including those they own and those they are selling
+  async getTicketsOfUserFromMultipleWalletAddresses(walletAddressList: string[]): Promise<MyTicketsDTO> {
+    try {
+      const ticketCollections = await this.ticketCollectionModel.find().exec();
```

Comment:
This will find every document, right? Is there a better operation?

Reply:
Good point. This one can take ticket collection id and just search from that ticket collection only. Thanks for pointing out.